CalibTracker/SiStripLorentzAngle/plugins/EnsembleCalibrationLA.cc

```diff
@@ -52,9 +52,9 @@ void EnsembleCalibrationLA::endRun(const edm::Run&, const edm::EventSetup& eSetu
 void EnsembleCalibrationLA::write_ensembles_text(const Book& book)
 {
   std::pair<std::string, std::vector<LA_Filler_Fitter::EnsembleSummary> > ensemble;
```

Comment:
doesn't this become an unused variable?
otherwise, there may be something that I miss in the scope/type resolution and this ```ensemble``` is used somehow in the loop as well.